src/engine/V3Eval.v3

```diff
@@ -588,6 +608,26 @@ def I8_POPCNT(a: u8) -> u8 {
 	}
 	return count;
 }
+def F32X4_PMIN(a: float, b: float) -> float {
+	if (a <= b) return a;
+	if (a > b) return b;
+	return a;
+}
+def F32X4_PMAX(a: float, b: float) -> float {
+	if (a >= b) return a;
+	if (a < b) return b;
+	return a;
+}
+def F64X2_PMIN(a: double, b: double) -> double {
```

Comment:
Can this be simplified to `if(a > b, b, a)`?

Reply:
Take this one for example:
```
def F32X4_PMAX(a: float, b: float) -> float {
	if (a >= b) return a; // line1
	if (a < b) return b; // line2
	return a; // line3
}
```
When either a or b is a NAN, the control flow will fall through to line 3. And we need to return the first operand here according to the test cases. (a and b could be a quiet NaN or a signaling NaN, having different bit patterns).

So I'm not able to simplify it to `return if(a >= b, a, b);` as the tests will fail.

Reply:
Take this one for example:
```
def F32X4_PMAX(a: float, b: float) -> float {
	if (a >= b) return a; // line1
	if (a < b) return b; // line2
	return a; // line3
}
```
When either a or b is a NAN, the control flow will fall through to line 3. And we need to return the first operand here according to the test cases. (a and b could be a quiet NaN or a signaling NaN as I observed).

So I'm not able to simplify it to `return if(a >= b, a, b);` as the tests will